core/src/iam/signin.rs

```diff
@@ -208,11 +222,12 @@ pub async fn db(
 			// Create the authentication key
 			let key = EncodingKey::from_secret(u.code.as_ref());
 			// Create the authentication claim
+			let exp = Some((Utc::now() + Duration::hours(1)).timestamp());
```

Comment:
So the timestamp isn't a session duration but a literal timestamp? Could we perhaps change this either to std time or instead have a session accessible "start time" that is not based on a calendar clock, but instead an always-incrementing value?

Reply:
In this implementation, the intended behavior is for the session to match the behavior of the token. We want the session to expire when the token does. If the system clock allows the token that was used to establish the session to be valid, the session should be valid as well. Otherwise, no increase in security would be achieved because even though the session would expire, the same token could be used to establish a new valid one.